deploy/package-operator/operator-kubeconfig/patches/kubeconfig-volume.yml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+    app.kubernetes.io/part-of: observability-operator
+    name: observability-operator
+spec:
+  template:
+    spec:
+      volumes:
+        - name: kubeconfig
+          secret:
+            defaultMode: 420
```

Comment:
how about 400 instead of 420 ? doesn't `2` mean allow allow write to group? `rwx`